src/py42/exceptions.py

```diff
@@ -153,7 +164,13 @@ def raise_py42_error(raised_error):
     HTTPError's response status code.
     """
     if raised_error.response.status_code == 400:
-        raise Py42BadRequestError(raised_error)
+        if json.loads(raised_error.response.text)[0]["name"] in [
+            "USER_IN_ACTIVE_LEGAL_HOLD",
+            "ACTIVE_LEGAL_HOLD",
```

Comment:
Could just `json.loads(raised_error.response.text)[0][u"name"].contains(u"ACTIVE_LEGAL_HOLD")` to encompass both

Reply:
I've moved all of this out of `py42.exceptions` based on your other comments. Looking back, it added overhead to any `400` response, which wasn't ideal.